### ISSUE TITLE
Both bundles end up being pretty large with this webpack.config.js

### ISSUE THREAD

**Issue by sstorie** [drewwyatt/ng2-webpack-starter#4]
First off, thanks for putting this out there. I liked the fact that the testing infrastructure was pretty simple, and not muddled up with a bunch of other (complicated) webpack stuff going on.

However, I have a question because I am running into a similar issue with my own webpack code. If I build the current version of the code I end up with both an angular2.bundle.js and app.bundle.js file...no problems there. However, each one is over 4MB in size:

```
C:\code\github\ng2-webpack-starter [master]> npm run build

> ng2-webpack-starter@0.0.1 build C:\code\github\ng2-webpack-starter
> webpack

ts-loader: Using typescript@1.8.9 and C:\code\github\ng2-webpack-starter\tsconfig.json
Hash: 3717a49b1b4072f9c49e
Version: webpack 1.12.14
Time: 4577ms
             Asset       Size  Chunks             Chunk Names
angular2.bundle.js    4.13 MB       0  [emitted]  angular2
     app.bundle.js    4.23 MB       1  [emitted]  app
        index.html  290 bytes          [emitted]
   [0] multi angular2 100 bytes {0} [built]
   [0] multi app 28 bytes {1} [built]
    + 539 hidden modules
Child html-webpack-plugin for "index.html":
        + 1 hidden modules
C:\code\github\ng2-webpack-starter [master]>
```

I was thinking the angular2 bundle could be relatively large, but the app bundle should be really small. Was that your intent by splitting the angular dependencies into a seperate bundle?

**Comment by sstorie** [comment]
I think with a small change to webpack.config.js this ends up producing what I was asking about:

```
var webpack = require('webpack');
var HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
    context: __dirname,
    entry: {
        'angular2': [
            'rxjs',
            'zone.js',
            'reflect-metadata',
            'angular2/platform/browser',
            'angular2/common',
            'angular2/core',
            'angular2/router',
            'angular2/http'
        ],
        'app': [
            './src/app/bootstrap'
        ]
    },
    output: {
        path: __dirname + '/dist',
        filename: '[name].bundle.js',
        sourceMapFilename: '[name].js.map',
        chunkFilename: '[id].chunk.js'
    },
    module: {
        loaders: [
            { test: /\.html$/,  loader: 'raw' },
            { test: /\.css$/,   loader: 'raw' },
            {test: /\.ts$/, loader: 'ts-loader'}
        ]
    },
    resolve: {
        extensions: ['','.ts','.js','.json']
    },
    plugins: [
        new HtmlWebpackPlugin({ template: 'src/index.html', chunksSortMode: 'none' }),
        new webpack.optimize.CommonsChunkPlugin("angular2", "angular2.bundle.js", Infinity)
    ],
    devServer: {
        port: 3000,
        host: "localhost",
        historyApiFallback: true,
        watchOptions: {
            aggregateTimeout: 300,
            poll: 1000
        }
  }
};
```

By utilizing the CommonsChunkPlugin you can properly split out the angular2 code into a seperate bundle, and reduce the app bundle considerably:

```
C:\code\github\ng2-webpack-starter [master +0 ~1 -0]> npm run build

> ng2-webpack-starter@0.0.1 build C:\code\github\ng2-webpack-starter
> webpack

ts-loader: Using typescript@1.8.9 and C:\code\github\ng2-webpack-starter\tsconfig.json
Hash: 38d716ffa3a55dec4e9e
Version: webpack 1.12.14
Time: 4554ms
             Asset       Size  Chunks             Chunk Names
angular2.bundle.js    5.51 MB       0  [emitted]  angular2
     app.bundle.js    3.06 kB       1  [emitted]  app
        index.html  290 bytes          [emitted]
   [0] multi angular2 112 bytes {0} [built]
   [0] multi app 28 bytes {1} [built]
    + 539 hidden modules
Child html-webpack-plugin for "index.html":
        + 1 hidden modules
```

**Comment by Healforgreen** [comment]
+1, although I am still getting a rather large 1.38MB for `app.bundle.js`.

**EDIT:** The OP already had this in, but I was missing `'angular2/platform/browser',` under `entry`.  Adding that removed it from `app.bundle.js` and placed it in `angular2.bundle.js`.

**Comment by drewwyatt** [comment]
Ah, sorry I'm just now getting to this! Thank you for bringing this up. I'll take a look this week.

**Comment by drewwyatt** [comment]
@sstorie @Healforgreen:  this was resolved with PR: #5
